app/api/rest_api.py

```diff
@@ -58,9 +58,10 @@ async def logout(self, request):
         await self.auth_svc.logout_user(request)
 
     async def landing(self, request):
+        """If user doesn't have access, server will attempt to redirect to login."""
```

Comment:
Are we trying to fill out docstrings for documentation (it seems so from some of the discussion on this ticket)?  Is so maybe a reword to make it sound less like a comment would be good, otherwise maybe just change it to a normal comment.

Reply:
Will change to normal comment